wolfcrypt/src/rsa.c

```diff
@@ -3150,7 +3163,7 @@ static int wc_RsaFunction_ex(const byte* in, word32 inLen, byte* out,
     !defined(NO_RSA_BOUNDS_CHECK)
     if (type == RSA_PRIVATE_DECRYPT &&
         key->state == RSA_STATE_DECRYPT_EXPTMOD) {
-
+        WOLFSSL_MSG("Calling RsaFunctionCheckIn...");
```

Comment:
Why have this?

Reply:
Same: It was a breadcrumb, seemed useful at the time, but I've removed it as I don't think it adds value at this time.